_config.yml

```diff
@@ -51,6 +51,10 @@ search:
   # Supports true or false (default)
   button: false
 
+# For copy button on code
+copy_button_on_code: true
+copy_button_on_code_time: 4000
```

Comment:
Renaming this variable `copy_button_on_code_timeout` could be more expressive and lead to fewer questions?